src/Jackett.Common/Indexers/HDTorrents.cs

```diff
@@ -144,21 +142,29 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                 };
                 var hasFreeleech = freeleechRanks.Contains(rank);
 
-                var rows = dom.QuerySelectorAll(".mainblockcontenttt > tbody > tr:has(a[href^=\"details.php?id=\"])");
+                var rows = dom.QuerySelectorAll("table.mainblockcontenttt tr:has(td.mainblockcontent)").Skip(1);
```

Comment:
Add skip in the next line, as in other trackers:
foreach (var row in rows.Skip(1))